client/src/app/pages/TrustRoot/components/Certificates.tsx

```diff
@@ -185,16 +185,22 @@ export const CertificatesTable: React.FC<ICertificatesTableProps> = ({ certifica
               <Tbody key={rowIndex} isExpanded={isCellExpanded(certificate)}>
                 <Tr key={rowIndex}>
                   <Td {...getSingleExpandButtonTdProps({ item: certificate, rowIndex })} />
-                  <Td modifier="breakWord">{certificate.issuer}</Td>
-                  <Td modifier="breakWord">{certificate.subject}</Td>
-                  <Td modifier="breakWord">{certificate.target}</Td>
-                  <Td width={10} modifier="truncate">
+                  <Td data-label="Issuer" modifier="breakWord">
```

Comment:
I wonder, is there any reason they are using `data-label` over the standard `data-testid`?